api/envoy/config/core/v3/address.proto

```diff
@@ -143,7 +142,6 @@ message Address {
 
     Pipe pipe = 2;
 
-    // [#not-implemented-hide:]
```

Comment:
we should have comments here and above explaining what the field is for?

Reply:
Done.